test/katello_test_helper.rb

```diff
@@ -88,7 +88,7 @@ module FixtureTestCase
 
     Setting::Content.load_defaults
 
-    @@admin = ::User.find(FIXTURES['users']['admin']['id'])
+    @@admin = ::User.unscoped.find(FIXTURES['users']['admin']['id'])
```

Comment:
i'm going to open a PR to move this to the top level rubocop.yml so that the behavior is consistent.